src/uprobes.d

```diff
@@ -9,6 +9,10 @@ provider julia {
     probe gc__sweep__end();
     probe gc__end();
     probe gc__finalizer();
+
+    probe rt__run__task(jl_task_t *task, jl_ptls_t ptls);
```

Comment:
How does this work? Does it not need the definitions of these structs?

Reply:
I haven't actually read fields from them, but I can definitely at least see the argument as an opaque pointer (which should be enough for basic task tracking).

Reply:
c.f. https://illumos.org/books/dtrace/chp-prog.html#chp-prog-5 and https://illumos.org/books/dtrace/chp-dtrace1m.html#chp-dtrace1m

On a Unix system with a real dtrace (not sure about the dtrace coming from systap) `#include julia.h` should work as long as you call with `-C` and `-I$include_dir `

Reply:
Yeah https://github.com/groleo/systemtap/blob/bd23cd846d787b59183d6d6a91046c1fe4cb3dc3/dtrace.in#L212 and there is also https://github.com/groleo/systemtap/blob/bd23cd846d787b59183d6d6a91046c1fe4cb3dc3/dtrace.in#L226